datafusion/core/tests/parquet/page_pruning.rs

```diff
@@ -495,22 +491,20 @@ async fn prune_f64_lt() {
 }
 
 #[tokio::test]
-#[ignore]
 async fn prune_f64_scalar_fun_and_gt() {
     // result of sql "SELECT * FROM t where abs(f - 1) <= 0.000001  and f >= 0.1"
     // only use "f >= 0" to prune
     test_prune(
         Scenario::Float64,
         "SELECT * FROM t where abs(f - 1) <= 0.000001  and f >= 0.1",
         Some(0),
-        Some(2),
+        Some(10),
```

Comment:
only use "f >= 0" to prune it will prune two pages (each 5 rows), so make it 10 make sense